### DISCUSSION TITLE
I am hearing no audio through my Multi-Output

### DISCUSSION THREAD

**Discussion by tenparks** [ExistentialAudio/BlackHole#610] [Q&A]
I was told to make a github request on this. I am not sure how to fix my issue. Here was the current thread of responses in emails. I'm still not sure what to do.

**Me:** Hello, I am currently having issues with my BlackHole 16 channel for mac. I'm not sure what to do, I used it once on stream then it hasn't worked since. All my settings are the same with Blackhole second to my primary audio source (my speakers or headphones). I've been searching online what's wrong and everything just says to have my primary audio source first which I have. The main issue is that when I put blackhole into my multi-output device I can't hear any of the audio being played or it's very slow and lagged. When I am only using blackhole as my main output my stream (I use streamlabs) can hear the audio on my computer, but I can not. And when I have blackhole in my multioutput device neither me nor my stream can hear it.
Do you have a way to fix this? I don't know what's wrong.

**Devin:** I just went through this with someone. Please follow this.
https://github.com/ExistentialAudio/BlackHole/issues/331

**Me:** Hello I have tried to reproduce the results but to no avail, since I've tried to use audacity in the same way they have but audacity has not updated to mac BigSur 11.2 is there any other way to fix my problem other than using audacity?

**Devin:** Send me a screenshot of your multi-output.

**Me:** My headphones currently are not plugged in so it is greyed out.
<img width="647" alt="Screen Shot 2021-03-06 at 3 12 37 PM" src="https://user-images.githubusercontent.com/82548859/114792543-9192d700-9d56-11eb-8f55-46b379ebbb2c.png">

**Devin:** Install the 2ch version. 

**Me:** So I downloaded the 2ch version, but it is very slow and lagged. Though I can now hear sound. I tried switching from the speakers and my airpods, but it is still slow and lagged.
<img width="652" alt="Screen Shot 2021-03-06 at 4 19 40 PM" src="https://user-images.githubusercontent.com/82548859/114792617-b38c5980-9d56-11eb-9c26-ba335ca71529.png">

**Devin:** You forgot to unselect the 16ch version. And select the 2ch version. 

**Me:** Do I have to uninstall the 16ch?

**Devin:** No

**Me:** I deselected 16ch and selected 2ch but I still am getting no audio. What should I do?
<img width="649" alt="Screen Shot 2021-03-06 at 4 51 36 PM" src="https://user-images.githubusercontent.com/82548859/114792702-d454af00-9d56-11eb-937c-facd9ff14459.png">

**Devin:** Make sure BlackHole 2ch is on the top of the list.
Set the sample rate to 48k.

**Me:** Like this? My 2ch is also at 48k, I am still not hearing anything.
<img width="652" alt="Screen Shot 2021-03-06 at 5 05 28 PM" src="https://user-images.githubusercontent.com/82548859/114792748-ecc4c980-9d56-11eb-8320-137a17327673.png">

**Comment by miba68** [comment]
Hello tenparks
do you still have the problem?
As I understand, a 48KHz speaker must appear as the first device in the "Out" list ... e.g. the MacBookPro speakers. You mute them in the midi controller.
Then you have the pods as the second device and can listen to the sound on them.
Hope this works :-)
